components/src/main/java/com/mercadolibre/android/andesui/feedbackinapp/AndesFeedbackInApp.kt

```diff
@@ -0,0 +1,219 @@
+        feedbackinappDivider = container.findViewById(R.id.andes_feedbackinapp_divider)
+        feedbackinappDivider.visibility = GONE
+        feedbackinappMessage = container.findViewById(R.id.andes_feedbackinapp_card_text)
+        feedbackinappMessage.gravity = Gravity.CENTER
+        action = container.findViewById(R.id.andes_feedbackinapp_button)
+    }
+
+    /**
+     * Sets a view id to this message.
+     *
+     */
+    private fun setupViewId() {
+        if (id == NO_ID) { // If this view has no id
+            id = View.generateViewId()
+        }
+    }
+
+    /**
+     * Gets data from the config and sets to the text component of this button.
+     *
+     */
+    private fun setupMessageComponent(config: FeedbackInAppConfiguration) {
+        if (config.cardText.isNullOrEmpty()) {
+            feedbackinappMessage.visibility = GONE
+        } else {
+            feedbackinappMessage.visibility = VISIBLE
+            feedbackinappMessage.text = config.cardText
+        }
+    }
+
+    private fun setupLabelComponent(config: FeedbackInAppConfiguration) =
+            if (config.cardText.isNullOrEmpty()) {
+                action.visibility = GONE
+            } else {
+                action.visibility = VISIBLE
+                action.text = config.action
+            }
+
+    fun setupLinkAction(text: String, onClickListener: OnClickListener) {
+        if (action.visibility == VISIBLE) {
+            buttonLabel = text
+            action.setOnClickListener(onClickListener)
+        } else {
+            when {
+                BuildConfig.DEBUG ->
+                    throw IllegalStateException("Cannot initialize a link action")
+                else ->
+                    Log.d("AndesMessage", "Cannot initialize a link action")
+            }
+        }
+    }
+
+    private fun setupShowtopdivider(config: FeedbackInAppConfiguration) {
+        if (config.showTopDivider) {
+            feedbackinappDivider.visibility = VISIBLE
+        } else {
+            feedbackinappDivider.visibility = GONE
+        }
+    }
+
+    private fun setupCardBackground(config: FeedbackInAppConfiguration) {
+        if (config.cardBackgroundColor.isNullOrEmpty()) {
+            this.setBackgroundColor(ContextCompat.getColor(context, R.color.andes_gray_070_solid))
+        } else {
+            this.setBackgroundColor(Color.parseColor(config.cardBackgroundColor))
+        }
+    }
+
+    @SuppressLint("RtlHardcoded")
+    private fun setupFeedbackInAppAlign(config: FeedbackInAppConfiguration) {
+        if (config.cardTextAlignment == AndesFeedbackInAppAlign.LEFT) {
+                    feedbackinappMessage.gravity = Gravity.LEFT
+        }
+        if (config.cardTextAlignment == AndesFeedbackInAppAlign.CENTER) {
+            feedbackinappMessage.gravity = Gravity.CENTER
+        }
+        if (config.cardTextAlignment == AndesFeedbackInAppAlign.RIGHT) {
+            feedbackinappMessage.gravity = Gravity.RIGHT
+        }
```

Comment:
Cambia la logica de 3 if por un when
```suggestion
        when(config.cardTextAlignment) {
               AndesFeedbackInAppAlign.LEFT -> feedbackinappMessage.gravity = Gravity.LEFT
               AndesFeedbackInAppAlign. RIGHT -> feedbackinappMessage.gravity = Gravity. RIGHT
               else -> feedbackinappMessage.gravity = Gravity.CENTER
        }
```